schemas/session-replay/browser/incremental-data-schema.json

```diff
@@ -28,6 +28,9 @@
     },
     {
       "$ref": "../common/viewport-resize-data-schema.json"
+    },
+    {
+      "$ref": "../common/pointer-interaction-data-schema.json"
```

Comment:
`PointerInteractionData` is added for browser even tho it will not be supported yet to prevent any conflicts with other potential `BrowserIncrementalData` additions. As a result PointerInteractionData will be source:9